synex-frontend/src/codebase/indexer.ts

```
@@ -1,14 +1,397 @@
+  }
+
+  // Extract class names
+  private extractClasses(content: string, extension: string): string[] {
+    const classes: string[] = [];
+
+    switch (extension) {
+      case '.js':
+      case '.jsx':
+      case '.ts':
+      case '.tsx':
+        const jsClassRegex = /class\s+([a-zA-Z_$][a-zA-Z0-9_$]*)/g;
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)